python_submitty_utils/submitty_utils/submitty_schema_validator.py

```diff
@@ -13,12 +13,14 @@ class SubmittySchemaException(Exception):
 
     def __init__(self, config_json, schema, message, title, schema_error):
         """Use to aid in printing helpful information about schema errors."""
-        super(SubmittySchemaException, self).__init__(schema_error.message)
+        error_message = schema_error.message if schema_error is not None else ''
+
+        super(SubmittySchemaException, self).__init__(error_message)
```

Comment:
In python 3, you can do `super().__init__(error_message)`.

Reply:
Updated in recent commit.